src/components/range-slider-label/range-slider-label.css

```diff
@@ -0,0 +1,22 @@
+@import "../../styles/constants.css";
```

Comment:
would this work in the context of other apps without explicit webpack config?

Reply:
you mean the constants.css file? mmm I think so but probably for webpack app will need css loader I guess...

Reply:
the `@import`, but i guess the browser should be able to resolve